src/main/java/com/jagrosh/jmusicbot/commands/dj/PlaynowCmd.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 John Grosh <john.a.grosh@gmail.com>.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jagrosh.jmusicbot.commands.dj;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.commands.PlayCommand;
+
+/**
+ *
+ * @author Daniel Liljeberg (liljebergxyz@gmail.com)
+ */
+public class PlaynowCmd extends PlayCommand
+{
+    public PlaynowCmd(Bot bot, String loadingEmoji)
+    {
+        super(bot, loadingEmoji);
+        this.name = "playnow";
+        this.help = "plays a single song now";
+    }
+    
+    @Override
+    public void doCommand(CommandEvent event)
+    {
+        if(event.getArgs().isEmpty() && event.getMessage().getAttachments().isEmpty())
+        {
+            event.replyWarning("Please include a song title or URL!");
+            return;
+        }
+        AudioHandler handler = (AudioHandler)event.getGuild().getAudioManager().getSendingHandler();
+        event.reply(event.getClient().getSuccess()+" Skipped **"+handler.getPlayer().getPlayingTrack().getInfo().title+"**");
```

Comment:
event.reply should only be used once, and we shouldn't assume that loading is going to succeed; this should be appended to the beginning of the edited-in message when we actually skip a track